src/projectscene/view/clipsview/selectionviewcontroller.cpp

```diff
@@ -24,7 +24,7 @@ void SelectionViewController::onPressed(double x, double y)
 
     if (!selectionController()->isDataSelected()) {
         TrackIdList tracks = determinateTracks(m_startPoint.y(), y);
-        selectionController()->setDataSelectedOnTracks(tracks, true);
+        selectionController()->setSelectedTracks(tracks, true);
     } else {
         setSelectionActive(false);
         selectionController()->resetDataSelection();
```

Comment:
BTW, you have a `::resetDataSelection` method in this class that does this, which you could probably get rid of.

Reply:
it's used on qml side so I can't remove it but it will be good to use it here instead of these two lines, good catch :D